web/app/models.py

```diff
@@ -226,7 +227,7 @@ def __str__(self):
 class TournamentPlayer(db.Model):
```

Comment:
gotcha. sucks that there has to be this duplication :(

I suppose  the assignment of the AGA id is the part we can't get around; otherwise we could just fake an entry and wait for the user to sign up to OR, the same as any other AGA user who hasn't yet signed up on OR.